cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2426,8 +2403,7 @@ Slice::Python::getImportFileName(const string& file, const UnitPtr& ut, const ve
 void
 Slice::Python::generate(const UnitPtr& un, bool all, const vector<string>& includePaths, Output& out)
 {
-    Slice::Python::MetadataVisitor visitor;
-    un->visit(&visitor);
+    validateMetadata(un);
```

Comment:
we should name the UnitPtr parameter unit.

Reply:
I fixed it in these locations, but in general we have many different (and inconsistent) names for unit.
`unt` is my personal favorite. We just have to save that `i` character...